src/css/styles.scss

```diff
@@ -24,29 +24,29 @@ h1 {
   margin: 0%;
   padding: 0.5% 0.7% 0%;
   width: 70%;
-};
+}
 
 h2 {
-  color: $recipe-type-color;
+  color: $recipe-font-color;
   margin: 10px 0px 10px 0px;
-};
+}
 
 h3 {
   color: $recipe-font-color;
   cursor: pointer;
   height: 17%;
   margin: 5px 5px 10px 0px;
   transition: .8s ease;
-};
+}
 
 h4 {
-  color: $recipe-font-color;
+  color: #092000;
```

Comment:
Love this color change. Much easier to see with higher contrast between text and button color.

Reply:
I also feel it was a great improvement from the previous color.